gcp-native-ts-k8s-ruby-on-rails-postgresql/infra/db.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright 2016-2021, Pulumi Corporation.  All rights reserved.
+
+import * as classic from "@pulumi/gcp";
+import * as gcp from "@pulumi/gcp-native";
+import { dbPassword, dbUsername, project } from "./config";
+
+// Provision a database for our Rails app.
+const instanceName = "pgdb";
+export const instance = new gcp.sqladmin.v1beta4.Instance("web-db", {
+    databaseVersion: "POSTGRES_9_6",
+    project: project,
+    name: instanceName,
+    instance:  instanceName,
+    settings: {
+        tier: "db-f1-micro",
+        ipConfiguration: {
+            authorizedNetworks: [{ value: "0.0.0.0/0" }],
+        },
+    },
+});
+
+// Create a user with the configured credentials for the Rails app to use.
+// TODO: Switch to gcp native version when User is supported.
```

Comment:
Do you know why we don't support it? Could you open an issue?

Reply:
The resource doesn't have a GET but there is a LIST to return all users. I cut https://github.com/pulumi/pulumi-gcp-native/issues/47 to track this family of resources.